tests/filecheck/mlir-conversion/with-mlir/dialects/emitc/emitc_ops.mlir

```diff
@@ -0,0 +1,17 @@
+// RUN: xdsl-opt %s | xdsl-opt | mlir-opt --allow-unregistered-dialect | filecheck %s
+
+//===----------------------------------------------------------------------===//
+// CallOpaqueOp
+//===----------------------------------------------------------------------===//
+
+%0 = "emitc.call_opaque"() {callee = "blah"} : () -> i64
+emitc.call_opaque "foo" (%0) {args = [
+  0 : index, dense<[0, 1]> : tensor<2xi32>, 0 : index
+]} : (i64) -> ()
+emitc.call_opaque "test" ()  : () -> ()
+
+// CHECK: builtin.module {
```

Comment:
Lets just omit the lines checking for `builtin.module`? (Especially as these are what is currently making the ci fail)

Reply:
ah, was trying to figure it out, locally it does not fail

Reply:
It's the mlir integration test that failed, so perhaps you don't have MLIR installed locally? (or perhaps a different mlir version)

Reply:
locally I'm on `LLVM version 20.1.1`, it fails if I remove the check for builtin.module

Reply:
Did you change the following line from `CHECK-NEXT` to `CHECK`?

Reply:
did not manage to get rid or it, but CI passes now with [bc831d7](https://github.com/xdslproject/xdsl/pull/4781/commits/bc831d75b8b71372ff07fa6a78e16d428ebceb24)